src/vs/workbench/contrib/terminalContrib/suggest/browser/terminal.suggest.contribution.ts

```diff
@@ -87,23 +86,19 @@ class TerminalSuggestContribution extends DisposableStore implements ITerminalCo
 		if (!enabled) {
 			return;
 		}
-		this.add(Event.runAndSubscribe(this._ctx.instance.onDidChangeShellType, async () => {
-			this._loadAddons(xterm.raw);
-		}));
-		this.add(this._contextKeyService.onDidChangeContext(e => {
-			if (e.affectsSome(this._terminalSuggestWidgetContextKeys)) {
+		this.add(Event.runAndSubscribe(this._configurationService.onDidChangeConfiguration, e => {
+			if (!e || e.affectsConfiguration(TerminalSettingId.SendKeybindingsToShell)) {
```

Comment:
Why does this matter `SendKeybindingsToShell` 🤔

Reply:
Oh, it was already like this before, just moved up

Reply:
I was almost going to remove this. I think it was added originally because ctrl+space on pwsh has a function, however we probably should remove it since if the user enables suggest they expect this to work.

Reply:
I can remove it

Reply:
done